perun-cli/assignGroupToResource

```diff
@@ -16,20 +16,21 @@ sub help {
 	--voId        | -v vo id
 	--voShortName | -V vo short name
 	--resource    | -r resource id
+	--async       | -a assigns group asynchronously
 	--batch       | -b batch
 	--help        | -h prints this help
 
 	};
 }
 
-my ($groupId, $groupName, $voId, $voShortName, $resourceId, $batch);
+my ($groupId, $groupName, $voId, $voShortName, $resourceId, $async, $batch);
```

Comment:
`assignInactive` and `autoAssignSubgroups` are missing

Reply:
Done.